src/components/DonationWidget/service.ts

```diff
@@ -1,7 +1,40 @@
 import { DonationMachineContext } from './machine/types';
 
+const DONATION_API_URL = 'http://membership.lvh.me:5000/donate.json';
```

Comment:
This will become a `.env`, I'm waiting to finish with @orlando the integration with membership and decide an approach for this one. _(I think we don't have env variables on this project yet.)_